Client/src/components/learning.tsx

```diff
@@ -0,0 +1,141 @@
+import '../css/learning.css';
+import theNut from '../assets/learning/nut.png';
+import treePic from '../assets/learning/tree.png';
+import grassPic from '../assets/learning/grass.png';
+import tallGrassPic from '../assets/learning/tall_grass.png';
+import { useNavigate } from 'react-router-dom';
+// import squirrel from '../assets/learning/squirrel.png';
+
+export default function LearningPage () {
+
+  const navigate = useNavigate()
+
+
+
+  return (
+    <>
+    <div className='initial-view'>
+      <div className='initial-view-text'>
+        <p>Welcome to the amazing learning experience.</p>
+        <p>There is going to be some more text here very soon.</p>
+        <p>For now, just scroll.</p>
+      </div>
+    </div>
+    <div className='all-algs'>
+      <div className='path-finding-algs'>
+        <h3>PATH FINDING ALGORITHMS</h3>
+        <div className='curve-1'>
+          <svg viewBox='-10 4 150 24'>
```

Comment:
Can this svg be given its own component?

Reply:
Done